charts/loki-distributed/templates/memcached-index-queries/poddisruptionbudget-memcached-index-queries.yaml

```diff
@@ -9,5 +9,7 @@ spec:
   selector:
     matchLabels:
       {{- include "loki.memcachedIndexQueriesSelectorLabels" . | nindent 6 }}
-  maxUnavailable: 1
+  {{- if not eq .Values.distributor.maxUnavailable null }}
```

Comment:
WDYT about using a `with` statement for these blocks? Might read a little cleaner than the comparison to null